### ISSUE TITLE
Include Groundtruth Alignment

### ISSUE THREAD

**Issue by M3ssman** [ulb-sachsen-anhalt/digital-eval#7]
# Description

Given the case, that groundtruth has been created once upon a time and the original(=referenced) image has been scanned again with different parameters (resolution, channels, size, angle, ... ) , the existing groundtruth would become unusable because of different geometric features. 

Currently, there's a clumsy way of achieving a somehow similar effect implemented by passing an extra argument `ignore_geometry`. But this make only the data "forget" it's coordinates, which on the other side requires additional manual efforts to extract the appropriate  image frame from the evaluation candidate an use only this part for ocr-ing as well.

Typical examples include newspaper digitalization when an image is first taken from microfilm/microfiche from which initial groundtruth got created and then shall be compared with a scan from original paper.

Requires the actual image to align existing groundtruth to it's actual geometry by calculating affine transformation matrix which handles translation, rotation and scaling. 

For sake of simple implementation see as classical optimization task, cf. remarks from [@bertsky](https://github.com/bertsky) on https://github.com/OCR-D/ocrd_segment/wiki/TemplateDrivenSegmentation) and try out the proposed algorithms. 

In real world scenarios like different image sizes the major problem is scaling, so the implementation should favor an approach that puts emphasis on this issue.

**Comment by bertsky** [comment]
Agreed! See also https://github.com/hnesk/ocr-experiments for a first demo
